From: Ben Hutchings <ben@decadent.org.uk>
Subject: tools/hv: Remove unused variables

Several local variables are unused and appear to be copy-pasta.
In kvp_update_file(), bytes_written is assigned but otherwise
unused.  We use ferror() to check for write errors, so this is
unneeded.

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
--- a/tools/hv/hv_kvp_daemon.c
+++ b/tools/hv/hv_kvp_daemon.c
@@ -125,7 +125,6 @@
 static void kvp_update_file(int pool)
 {
 	FILE *filep;
-	size_t bytes_written;
 
 	/*
 	 * We are going to write our in-memory registry out to
@@ -140,9 +139,8 @@
 		exit(EXIT_FAILURE);
 	}
 
-	bytes_written = fwrite(kvp_file_info[pool].records,
-				sizeof(struct kvp_record),
-				kvp_file_info[pool].num_records, filep);
+	fwrite(kvp_file_info[pool].records, sizeof(struct kvp_record),
+	       kvp_file_info[pool].num_records, filep);
 
 	if (ferror(filep) || fclose(filep)) {
 		kvp_release_lock(pool);
@@ -206,7 +204,7 @@
 }
 static int kvp_file_init(void)
 {
-	int ret, fd;
+	int fd;
 	FILE *filep;
 	size_t records_read;
 	char *fname;
@@ -328,7 +326,6 @@
 				 const char *value, int value_size)
 {
 	int i;
-	int j, k;
 	int num_records;
 	struct kvp_record *record;
 	int num_blocks;
